src/test/opentrackIO_parser/python/README.md

```diff
@@ -0,0 +1,43 @@
+README.txt
+
+Documentation for opentrackIO_parser.py
```

Comment:
You can use #'s for headings and ```<code>``` for code to prettify the markdown.

Reply:
That didn't escape properly - I mean 3 reverse ticks!

Reply:
Since this is a fork and not a branch, I will merge this PR and then do this on my branch